datafusion/core/tests/sqllogictests/test_files/timestamps.slt

```diff
@@ -1031,7 +1031,7 @@ SELECT '2000-01-01T00:00:00'::timestamp - '2010-01-01T00:00:00'::timestamp;
 0 years 0 mons -3653 days 0 hours 0 mins 0.000000000 secs
 
 # Interval - Timestamp => error
-statement error DataFusion error: Error during planning: interval can't subtract timestamp/date
+statement error DataFusion error: Error during planning: Invalid interval subtraction: Interval\(MonthDayNano\) \- Timestamp\(Nanosecond, None\)
```

Comment:
I would like to hear opinions on this new message. I think it is better but still not great as it has references to the detailed Arrow types.